### ISSUE TITLE
What to install for mail delivery?

### ISSUE THREAD

**Issue by ModProg** [steadfasterX/arch_uau#4]
I installed `ssmtp` as recommended by the AUR package, it did not work.

**Comment by ModProg** [comment]
the answer is `s-nail`.

**Comment by steadfasterX** [comment]
whatever works. I use [ssmtp](https://aur.archlinux.org/packages/ssmtp) (which is mentioned in the [README](https://github.com/steadfasterX/arch_uau-pkg/blob/master/README.md), too) though.

"it didnt work" is well what should I say? That comment is not helpful in any way. Explain what does not work, share configs, share logs, .... will close that issue but feel free to open an issue with providing the minimum things needed to get help.

**Comment by ModProg** [comment]
I had `ssmtp` installed but got this error message:
```
fetching the latest news (and AUR comments) before upgrading
starting the unattended system upgrade!
check & upgrade repo binaries
check & upgrade AUR packages
/usr/local/bin/uau: Zeile 67: mail: Kommando nicht gefunden.
finished
[archupdater@desktop-mp modprog]$ whereis ssmtp
ssmtp: /usr/bin/ssmtp /etc/ssmtp /usr/share/man/man8/ssmtp.8.gz
[archupdater@desktop-mp modprog]$
```

**Comment by steadfasterX** [comment]
omg.. sorry, yea.. s-nail.. I totally forgot about that one :'D thx again for pointing

**Comment by steadfasterX** [comment]
fixed in [2.1.0-2](https://aur.archlinux.org/cgit/aur.git/commit/?h=uau&id=16151bc684a617954d78672706f40444dec0b3ba)
